src/datalist-ajax.js

```diff
@@ -186,10 +186,22 @@ class AutoComplete extends HTMLElement {
         for (let d = 0; d < data.length && optMax > 0; d++) {
 
           const value = this.resultname ? data[d][this.resultname] : data[d];
+          const res = [];
+          if (this.storekeys){
+              let keys = this.storekeys.split(',');
+              for (let k of keys) {
+                  if (k in data[d]){
+                    res[k] = data[d][k];
+                  }
+              }
+          }
```

Comment:
Will this work for nested properties that you support elsewhere, e.g. `storekeys="name.first, name.last"`?

Reply:
ive change this, now it works,